src/routes/oath.ts

```diff
@@ -27,19 +28,22 @@ const SAASQUATCH_CLIENT_SECRET = process.env.SAASQUATCH_CLIENT_SECRET;
 
 export const tokenStore: any = {};
 
-export const isAuthorized = (userID: string) =>{
-	if(userID == undefined)
-	{
-		return false
-	}
-	else
-	{
-		return tokenStore["userID"] == userID ? true : false;
+// Generate Server Access Token for frontend requests
+const generateAccessToken = (userID: string) => {
+	return jwt.sign({
+		data: userID
+	}, process.env.SERVER_TOKEN_SECRET, { expiresIn: '1h' });
```

Comment:
Do we think 1 hour is sufficient time for the cookie to have an active login?

Reply:
Yes